### DISCUSSION TITLE
Inability to get testnet Ether from various Sepolia faucets

### DISCUSSION THREAD

**Discussion by MichealAkinleye** [Cyfrin/foundry-full-course-cu#2454] [General]
Please guys, I have been unable to get some ether to get started with writing even my first smart contra t as a beginner. I got to know that Sepolia testnet is PoS, and that it's required that my mainnet wallet contains some ether which I do not have as a total beginner. I'm beginning to feel discouraged. Please can anyone help me.out?
Here's my Sepolia test net wallet address:
0x6f704164654441619ff7ED9134EBd1970033ff2D

**Comment by EngrPips** [comment]
Hello @MichealAkinleye, Please use this chainlink faucet https://faucets.chain.link/sepolia. It does not require any balance on the mainnet.

**Reply by MichealAkinleye** [reply]
Okay. I'll give it a try. Thanks.

**Reply by EngrPips** [reply]
Yeah, Please do.

**Reply by EngrPips** [reply]
> I want to ask whether this will be on the main network or not.

You mean want a faucet that will give you free ether on mainnet?

**Reply by EngrPips** [reply]
What do you mean by `without going online`?

**Reply by EngrPips** [reply]
LMAOO

**Reply by may-tonk** [reply]
> Hello @MichealAkinleye, Please use this chainlink faucet https://faucets.chain.link/sepolia. It does not require any balance on the mainnet.您好，请使用 Chainlink 水龙头 https://faucets.chain.link/sepolia 。它不需要主网余额。

It's still not to work 
Requests for native tokens require the user to hold at least 1 LINK on Ethereum Mainnet. This restriction does not apply to requests for testnet LINK.

**Reply by vijay-softment** [reply]
You can claim 1 sepolia eth per day from - https://softment.com/sepolia-faucet

**Comment by Neddie28** [comment]
I'm having the same issue in getting arbitrum sepolia testnet while going the chainlink ccip course please can someone help me with some:
0x8fcd4D74d15EEf8d7689C180c2ABcb4071130314

**Reply by EngrPips** [reply]
I answered you on Discord

**Comment by DEX-010** [comment]
I am actually having problems with bridging on zksync, I was able to get faucet tokens on gcp, but I am required to have a certain amount of ethereum on mainnet, I am quite confused

**Reply by EngrPips** [reply]
You mean that to be able to bridge the bridge, you have to have some Ethereum on the mainnet?

**Reply by DEX-010** [reply]
Yeah, thanks I have been able to solve that already! Currently I am having problems deploying on zksync from remix. I have bridged to zksync testnet already, now after compiling using zksync plug in and connecting wallet to the deployment platform. I hit the deployment button rather than a pop-up window from metamask asking for me to sign, nothing happens
In the console, it just shows "deploying simple_storage" and nothing happens otherwise. I have been stuck on this all day

**Reply by EngrPips** [reply]
Refresh your Remix page, clear your browser cache, and then retry

**Comment by mdsumair** [comment]
Can anyone send me sepolia eth testnet tokens on my addresses 0x806419Cb1954E42f51F9aBc0f4456Cb26780b86C

**Reply by EngrPips** [reply]
Try this https://cloud.google.com/application/web3/faucet/ethereum/sepolia or this https://sepolia-faucet.pk910.de/

**Reply by may-tonk** [reply]
> Try this https://cloud.google.com/application/web3/faucet/ethereum/sepolia or this https://sepolia-faucet.pk910.de/试试这个 https://cloud.google.com/application/web3/faucet/ethereum/sepolia 或者这个 https://sepolia-faucet.pk910.de/

It's still no use
Do you have any other faucets?
This my addresses 0x03821460938885DCDBe111236A2da58607aB276D can anyone help me

**Reply by EngrPips** [reply]
If I find one that may be useful, I will share it with you.

**Reply by may-tonk** [reply]
> If I find one that may be useful, I will share it with you.如果我发现一个可能有用的，我会与你分享。

Thanks

**Reply by vijay-softment** [reply]
You can claim 1 sepolia eth per day from - https://softment.com/sepolia-faucet

**Comment by xujiewocao** [comment]
I encountered the same situation.

**Reply by xujiewocao** [reply]
my transaction address: 0xA7a2BC63eCAcA40D546F051B56811EAe2B0376Df

**Reply by vijay-softment** [reply]
You can claim 1 sepolia eth per day from - https://softment.com/sepolia-faucet

**Comment by Samaa-Ghorab** [comment]
This was very helpful to me. it gives you 0.05 Sepolia ETH every 24 hours : https://cloud.google.com/application/web3/faucet/ethereum/sepolia

**Reply by EngrPips** [reply]
Indeed.

**Reply by vijay-softment** [reply]
You can claim 1 sepolia eth per day from - https://softment.com/sepolia-faucet

**Reply by Samaa-Ghorab** [reply]
@vijay-softment yes it worked and I wasn't have to wait long mining time to receive the ETH. Thanks a lot
